src/applications/caregivers/routes.jsx

```diff
@@ -7,7 +7,7 @@ const route = {
   component: App,
   indexRoute: { onEnter: (nextState, replace) => replace('/introduction') },
 
-  childRoutes: createRoutesWithSaveInProgress(formConfig),
+  childRoutes: createRoutes(formConfig),
```

Comment:
If you do this, save in progress won't work.

That is, you won't have the `form-saved`, `error`, or `resume` routes defined, and the page component will be `FormPage` instead of `RoutedSavablePage`, which means it won't have the save link, nor will it auto-save the form.

Reply:
Well the point if this PR is to remove save-in-progress. I dont really understand what `error ` and `resume` routes do though...

Reply:
Why do you want to remove save-in-progress functionality?

Reply:
@bkjohnson it is not apart of our Acceptance Criteria I forget the exact reason but I cant recall of the top of my head @ala-yna can you answer this?

Reply:
The reason being is due to Legal at the VA since we are acquiring certifications if the user is logged in and they sign for other parties VA sees that as the person being logged in signing for other people.